styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -325,7 +325,8 @@ private static WorkflowInstance getWorkflowInstance(Namespace namespace) {
   private static class StyxCliParser {
 
     final ArgumentParser parser = ArgumentParsers.newArgumentParser("styx")
-        .description("Styx CLI");
+        .description("Styx CLI")
+        .version("styx " + Main.class.getPackage().getImplementationVersion());
```

Comment:
Not sure about the dash. I think it should either match the program name (`$0`) or be apparent that it's not the same. `Styx CLI 0.2.2`?